packages/babel-plugin-proposal-class-properties/src/index.js

```diff
@@ -75,8 +79,33 @@ export default function(api, options) {
     inherits: syntaxClassProperties,
 
     visitor: {
+      ExportDefaultDeclaration(path) {
```

Comment:
this is literally copied from [here](https://github.com/babel/babel/blob/62bbee97d7245b897973d290d1700797a85fe168/packages/babel-plugin-transform-classes/src/index.js#L16-L31) any suggestions what's the best way to dedupe this?

Reply:
You can be a `path.splitExportDeclaration()` function (or a `@babel/helper-...`)

Reply:
was thinking about both, weren't sure which one to pick :D I'd probably go with latter, thanks

Reply:
https://github.com/babel/babel/blob/master/packages/babel-traverse/src/scope/lib/renamer.js#L42-L85 looks similar